quicksort/quickSort.js

```diff
@@ -0,0 +1,41 @@
+/*
+*	Implementation of quicksort algorithm
+*	Time complexity: O(nlogn)
+*	Space complexity: O(n)
+*/
+function partition (arr, start, end) {
+	let pivot = arr[end];
```

Comment:
Please randomize pivot, as its done in python implementation. You didn't randomize it.